### PR TITLE
Add a new 'Added on' column into the payment methods table.

### LINKED ISSUE #1691: Add a created_at column to the payment methods table

### PR DESCRIPTION
**Pull request by ttfra**
The new column is added to the PaymentMethods table which will include creation date information in the following format: `Added on <Month>, <dd>,  <YYYY>`

Fixes #1691.

Screenshot 2024-08-09 at 17 03 01

### CODE REVIEW

**Comment by ttfra** [comment]
I have read the CLA Document and I hereby sign the CLA

File: Gemfile.lock [hunk 1]
```diff
@@ -330,6 +330,7 @@ PLATFORMS
   arm64-darwin-22
   arm64-darwin-23
   x86_64-darwin-20
+  x86_64-darwin-23
```
Comment (enescakir) [inline Gemfile.lock:333]: It seems that no one in our team uses macOS 14 with an Intel-based processor. Could you make this change in a separate commit?

Comment (ttfra) [reply Gemfile.lock:333]: Thanks, done

**Review by enescakir — APPROVED** [review]
Thanks for the contribution 🚀
